x/mint/types/params_test.go

```diff
@@ -0,0 +1,373 @@
+package types
+
+import (
+	"github.com/ignite/modules/testutil/sample"
+	"math/rand"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParamsValidate(t *testing.T) {
+	invalidInflationMax := DefaultParams()
```

Comment:
I think it is be more visual to define the type directly in the test case in the table and keep initialization as minimal as possible